notebooks/wavs.ipynb

```diff
@@ -0,0 +1,236 @@
+{
```

Comment:
This can be more easily reviewed on: https://app.reviewnb.com/uw-ssec/uwloc-db/pull/1/